pkg/storage/gcs/gcp_labels_tags.go

```
@@ -312,46 +314,63 @@ func (t *tagServiceManager) addTagsToStorageBucket(ctx context.Context, cr *imag
 
-	return opts, nil
+	return opts, ud, nil
 }
 
 // getTagBindingsClient returns the client to be used for creating tag bindings to
 // the resources.
-func getTagBindingsClient(ctx context.Context, listers *regopclient.StorageListers, location string) (TagBindingsService, error) {
-	endpoint := fmt.Sprintf("https://%s-%s", location, resourceManagerHostSubPath)
-	opts, err := getTagClientOptions(listers, endpoint)
+func getTagBindingsClient(ctx context.Context, listers *regopclient.StorageListers, location string) (TagBindingsService, string, error) {
+	opts, ud, err := getTagClientOptions(listers, location)
```

golangci-lint (2.12.2)

[error] Function `getTagClientOptions` should pass the context parameter

(contextcheck)